spec/appliance_console/prompts_spec.rb

```diff
@@ -1,17 +1,27 @@
-require "stringio"
+require 'stringio'
+require 'tempfile'
+require 'readline'
 require "appliance_console/errors"
 require "appliance_console/prompts"
 require "active_support/all"
 require 'highline/import'
 require "linux_admin"
 
 describe ApplianceConsole::Prompts do
-  let(:input)  { StringIO.new }
+  let(:input) do
+    temp_stdin = Tempfile.new("temp_stdin")
+    File.open(temp_stdin.path, 'w+')
```

Comment:
Does this not work if we just use the `Tempfile` object directly? Why reopen it as a `File` type?